frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -167,8 +172,8 @@ export class WebsocketConnection {
    */
   private wsConnectionTimeoutId?: number;
 
-  public constructor(args: Args) {
-    this.args = args
+  public constructor(props: Props) {
```

Comment:
I originally called this `args` to differentiate from React's `props`. Otherwise it's easy to forget that this is not a React component. So I think `args` is still clearer.